packages/dd-trace/src/appsec/blocked_templates.js

```diff
@@ -97,3 +100,18 @@
 </body>
 
 </html>
+`
+
+const json = `{
```

Comment:
totally optional idea: if you wanted, you could stringify at runtime:

```js
const json = JSON.strinigfy({
  errors: {
    ...
  }
})
```

That way we get the guarantee that the JSON is valid.